exercises/practice/acronym/zcl_acronym.clas.testclasses.abap

```diff
@@ -0,0 +1,89 @@
+*"* use this source file for your ABAP unit test classes
+CLASS ltcl_acronym DEFINITION FINAL FOR TESTING
+  DURATION SHORT
+  RISK LEVEL HARMLESS.
+
+  PRIVATE SECTION.
+    DATA cut TYPE REF TO zcl_acronym.
+
+    METHODS setup.
+    METHODS:
+      basic_phrase FOR TESTING,
+      lowercase_words FOR TESTING,
+      punctuations FOR TESTING,
+      all_caps FOR TESTING,
+      punc_wo_space FOR TESTING,
+      long_phrase FOR TESTING,
+      consecutive_delimeters FOR TESTING ,
+      apostrophes FOR TESTING,
+      underscore FOR TESTING,
+      empty_string FOR TESTING.
+ENDCLASS.
+
+
+CLASS ltcl_acronym  IMPLEMENTATION.
+  METHOD setup.
+    cut = NEW zcl_acronym( ).
+  ENDMETHOD.
+
+  METHOD basic_phrase.
+    cl_abap_unit_assert=>assert_equals(
+       act = cut->parse( `Portable Network Graphics`)
```

Comment:
please align the use of whitespace

Reply:
thanks, made changes to all the calls and aligned the whitespaces.